### PR TITLE
[Hotfix] Change page after autofill based on type of listing #165574739

### PR DESCRIPTION
**Pull request by akegan**
https://www.pivotaltracker.com/story/show/165574739

### CODE REVIEW

File: app/assets/javascripts/short-form/templates/b0-autofill-preview.html.slim [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
   .button-pager
     .button-pager_row.primary
-      button.button.primary ui-sref="dahlia.short-form-application.name" ng-click="trackAutofill()"
+      button.button.primary ng-click="trackAutofill()"
```
Comment (chrisdolendo) [inline app/assets/javascripts/short-form/templates/b0-autofill-preview.html.slim]: This should be `startAutofilledApp` yeah?

File: app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee [hunk 2]
```diff
@@ -75,8 +75,16 @@ ShortFormApplicationController = (
 
   $scope.emailRegex = SharedService.emailRegex
 
-  $scope.trackAutofill = ->
+  $scope.startAutofilledApp = ->
     AnalyticsService.trackFormSuccess('Application', 'Start with these details')
+    # Start page after autofill preview
+    $scope.go($scope.getFirstFillableAppPage())
+
+  $scope.getFirstFillableAppPage = ->
```
Comment (chrisdolendo) [inline app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee]: I noticed that this function already exists in `ShortFormNavigationService` which is already injected in this controller and you can just call to return the right statename...

```
  Service.initialState = () ->
    if ListingIdentityService.isSale(ShortFormApplicationService.listing)
      'dahlia.short-form-application.prerequisites'
    else
      'dahlia.short-form-application.name'
```

Comment (akegan) [reply app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee]: fixed!

**Comment by akegan** [comment]
it's calling web-prod to check for semaphore failure. web-full is passing so I'm going to merge

**Pull request merged**